src/UnityExtension/Assets/Editor/GitHub.Unity/UI/ProjectWindowInterface.cs

```diff
@@ -26,7 +26,7 @@ class ProjectWindowInterface : AssetPostprocessor
         private static CacheUpdateEvent lastRepositoryStatusChangedEvent;
         private static CacheUpdateEvent lastLocksChangedEvent;
         private static IRepository Repository { get { return manager != null ? manager.Environment.Repository : null; } }
-        private static bool IsInitialized { get { return Repository != null && Repository.CurrentRemote.HasValue; } }
+        private static bool IsInitialized { get { return Repository != null; } }
```

Comment:
This check is used to determine whether to show lock/unlock menus. If you're going to change this logic, you need to also fix how we enable those menus, because we can't lock/unlock if there's no remote.